modules/features/discover/src/main/res/layout/collection_header.xml

```diff
@@ -0,0 +1,63 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:layout_marginVertical="8dp"
+    android:layout_marginLeft="8dp"
+    android:clickable="true"
+    android:focusable="true">
+
+    <com.google.android.material.card.MaterialCardView
+        android:layout_width="179dp"
+        android:layout_height="match_parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageView
+            android:id="@+id/imageHeader"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="?attr/defaultArtwork"
+            android:scaleType="centerCrop"
+            tools:ignore="ContentDescription" />
```

Comment:
On first load the view seems to larger than the row height. 
<img width="458" alt="Screenshot 2025-03-12 at 2 50 41 pm" src="https://github.com/user-attachments/assets/b718486f-b569-4bb3-b064-4f127c973040" />